crates/api_models/Cargo.toml

```diff
@@ -35,7 +35,7 @@ url = { version = "2.5.0", features = ["serde"] }
 utoipa = { version = "4.2.0", features = ["preserve_order", "preserve_path_order"] }
 frunk = "0.4.2"
 frunk_core = "0.4.2"
-
+diesel_models = { version = "0.1.0", path = "../diesel_models" }
```

Comment:
Ideally this import shouldn't be there.
we could instead move these enums to common_enums maybe

Reply:
Done